### ISSUE TITLE
esp32 is available miracl？

### ISSUE THREAD

**Issue by qaz1423078740** [miracl/MIRACL#116]
I want to implement bilinear pairing in esp32.But I did not find the corresponding library. Can this library be used in esp? How do you do that?

**Comment by mcarrickscott** [comment]
Yes, we have implemented bilinear pairing on an esp32. See arduino
directory in https://github.com/miracl/core

**Comment by qaz1423078740** [comment]
Can only generate one? I want to use share with a bilinear pairing, what do I do? Also, I used 46.Kyber and ran the routine with the following error   
C:\Users\14230\Documents\Arduino\libraries\share\eddsa.cpp:28:10: fatal error: eddsa_ZZZ.h: No such file or directory
 #include "eddsa_ZZZ.h"
          ^~~~~~~~~~~~~
compilation terminated.
exit status 1
I checked eddsa.h and found that EDDSA_ZZZ_H was defined. Is it a problem of definition.


eddsa.h:
#ifndef EDDSA_ZZZ_H
#define EDDSA_ZZZ_H

#include "ecp_ZZZ.h"
And the generated file does not have ecp ZZZ.h

**Comment by qaz1423078740** [comment]
![image](https://github.com/miracl/MIRACL/assets/92016538/17395663-3a0d-445a-a23c-5261cb312899)
This is the directory I selected 46 to generate

**Comment by qaz1423078740** [comment]
![image](https://github.com/miracl/MIRACL/assets/92016538/caf11cf5-1063-4cdb-981f-ed4185a4b099)
This is the directory I selected 46 to generate

**Comment by mcarrickscott** [comment]
Choose the pairing friendly curve that you want to use, for example 28.
BN254

Script was not updated for recent addition, should be OK now, so download
again.

Mike

**Comment by qaz1423078740** [comment]
thanks
